fabbwled-backend/src/main/resources/sections/433-468.yaml

```diff
@@ -0,0 +1,132 @@
+        and other services the temple can offer. To qualify as an initiate you must have a COMBAT score of at least 6.
+        You cannot become an initiate of Tyrnai if you are already an initiate of another temple.
+        If you choose to become an initiate (and meet the qualification) write Tyrnai in the God box on your AdventureSheet.
+    - turnTo: 526
+- number: 436
+  content:
+    - text: |
+        You find yourself washed up on a rocky shore, battered and cold, but lucky to be alive. 
+        You haul yourself up the beach on to land. Fortunately to you, you are near the Trading Post on the Isle of Druids.
+    - turnTo: 195
+- number: 437
+  content:
+    - text: |
+        Becoming an initiate of Sig gives you the benefit of paying less for blessings and other services.
+        Also, you can add 1 to your THIEVERY score, as Sig will watch over your pilfering activities and keep you safe from the law.
+        It costs 50 Shards to become an initiate. You cannot become an initiate if you are already an initiate of any other temple.
+        If you choose to become an initiate. Write Sig in the Godbox on your Adventure Sheet – and cross off 50 Shards.
+    - turnTo: 235
+- number: 438
+  content:
+    - text: |
+        The climb is too arduous for you. The air becomes too thin to breathe,
+        and you cannot find the footholds you need to pull yourself up.
+        You are forced to abandon the attempt and climb back down.
+    - turnTo: 244
+- number: 439
+  content:
+    - text: Your ship is sailing in the coastal waters beside Yellowport.
+    - choice:
+        - text: Sail into Yellowport Harbour
+          turnTo: 555
+        - text: Sail north east towards Scorpion Bight
+          turnTo: 430
+        - text: Sail south west along the coast
+          turnTo: 120
+        - text: Sail south into the Violet Ocean
+          turnTo: 366
+- number: 440
+  content:
+    - text: |
+        Becoming an initiate of Elnir gives you the benefit of paying less for blessings and other services the Temple can offer.
+        It costs 60Shards to become and initiate. You cannot become an initiate of Elnir if you are already an initiate of another temple.
+        If you choose to become an initiate, write Elnir in the Godbox on your Adventure Sheet – and then cross 60 shards off your Adventure Sheet.
+    - turnTo: 568
+- number: 441
+  content:
+    - text: |
+        You clamber up on to the battlements of the Temple of Tyrnai with relative ease and manoeuvre yourself
+        into a position overlooking the golem guards at the gate. Now you have to reach down without being noticed.
+    # TODO: Add dice functionality
+- number: 442
+  content:
+    - text: |
+        You set out to explore the city of Yellowport at night,unwholesome though it is with its reeking air and dusty ochrestreets.
+    - choice:
+        - text: Head for the rough part of the town
+          turnTo: 21
```

Comment:
same here

Reply:
yeah not easy if the schema validation doesn't help you